Create documentation_process.md (#11)
research-software-ecosystem/micoreca: doc/documentation_process.md

# Curation processes

## Curation of the workflows

To have the most relevant workflows to the microbiome community, we manually curated the workflows extracted from WorkflowHub and programmatically filtered using the keywords defined. 
To do so, we excluded the workfows where the description evolved around terms like: 
- Single genome analysis  
- whole genome sequencing 
- single-end ChIP-Seq data
- reference genome annotation
- biomedical
- Human
- rare diseases 
- NGS SARS-Cov2 samples
- Calculation not related to the microbiome (LU matrix factorization on a sparse blocked matrix)
- Images and Image reconstruction 
- molecular network

> The workflows that were retrieved and kept if corresponding to one or more of the following steps for microbiome analysis.

Reviewer: ```suggestion
The extracted resources are kept if they correspond to one or more of the following steps for microbiome analysis:
```
